tools/js-sdk-release-tools/src/llc/apiVersion/apiVersionTypeExtractor.ts

```diff
@@ -14,12 +55,18 @@ export const getApiVersionType: IApiVersionTypeExtractor = async (
         const typeFromClient = await getApiVersionTypeFromRestClient(packageRoot, pattern, tryFindRestClientPath);
         if (typeFromClient !== ApiVersionType.None) return typeFromClient;
     }
-    
-    const parametersPath = join(packageRoot, "src/parameters.ts");
-    if (!(await exists(parametersPath))) {
-        throw new Error(`Failed to find parameters file '${parametersPath}'.`);
+
+    logger.info('No client found, fallback to get api version type in operation\'s parameter');
```

Comment:
sorry i am confused, you are trying to get the sdk version or service api version?

Reply:
`api-version` in http request